docs/csharp/language-reference/index.yml

```
@@ -31,7 +31,9 @@ landingContent:
     linkLists:
       - linkListType: whats-new
         links:
-          - text: "What's new in C# 13"
+          - text: "What's new in C# 14"
+            url: ../whats-new/csharp-14.md
+            - text: "What's new in C# 13"
```

This format error is breaking the build:

```suggestion
          - text: "What's new in C# 13"
```